bin/generate

```diff
@@ -2,19 +2,22 @@
 'use strict';
 
 var Bluebird = require('bluebird');
+var Path     = require('path');
 
 var CLI       = require('../lib/cli');
 var Changelog = require('../lib');
 var File      = require('../lib/file');
 
 CLI.parse(process.argv);
 
+var filePath = Path.join(CLI.subDirectory, CLI.file);
```

Comment:
Hmm, I'm not if we can join the paths together here since some people might be passing in `-` for stdout. We might need to pass `CLI.subDirectory` through to the two functions. We also should probably be checking for `-` explicitly in [`File.readIfExists`](https://github.com/lob/generate-changelog/blob/797909e6c0528491d5c62efc39ab531fd4c4e41e/lib/file.js#L22-L27), but you don't have to do that in this PR.